### PR TITLE
Added website,blog,irc functionalities

### PR DESCRIPTION
**Pull request by sidntrivedi**
@vaibhavk Tried to implement the intro functionality through which the bot gives intro whenever it joins the telegram group. But, not able to find out where to call the intro function. Have called other functions in the same switch case.

### CODE REVIEW

**Comment by vnzongzna** [comment]
I've been refereeing to [this](https://github.com/go-telegram-bot-api/telegram-bot-api) repository.
You can find the function [here](https://godoc.org/github.com/go-telegram-bot-api/telegram-bot-api#NewChatAction)

File: main.go [hunk 1]
```diff
@@ -9,6 +9,9 @@ import (
 
 var bot *tbot.BotAPI
 
+func intro(ID int64){
```
Comment (vnzongzna) [inline main.go]: Remove the functions that aren't usable right now. Go doesn't allow function definition that weren't used at all.

Comment (sidntrivedi) [reply main.go]: Ok.

**Comment by sidntrivedi** [comment]
@vaibhavk PTAL now. Will look into how to implement the intro part.

**Pull request merged**